docs/getting-started.md

```diff
@@ -569,8 +626,9 @@ the new commit.
 
     <h3>Allow Gitpod to be able to push your changes to GitHub</h3>
     
-    1. When logged into Gitpod, visit their [Settings
-       page](https://gitpod.io/settings).
+    1. When logged into Gitpod, use [this link](https://gitpod.io/settings) 
```

Comment:
again, this doesn't quite match the hyperlink style we've tried to stick to elsewhere